selene/src/roblox/base.toml

```
@@ -622,15 +626,19 @@ required = false
 
 [[UDim2.fromOffset.args]]
 type = "number"
+required = false

```

It doesn't feel useful to me to use `UDim2.fromOffset()` for an empty UDim2 when we have UDim2 for that purpose.

```suggestion
required = "use UDim2.new() if you want an empty UDim2"
```